java/test/org/openqa/selenium/testing/JupiterTestBase.java

```diff
@@ -43,6 +45,20 @@ public abstract class JupiterTestBase {
 
   @RegisterExtension protected static SeleniumExtension seleniumExtension = new SeleniumExtension();
 
+  static {
+    if ("true".equalsIgnoreCase(System.getenv("DEBUG"))) {
+      Logger rootLogger = Logger.getLogger("");
+      rootLogger.setLevel(Level.FINE);
+      Arrays.stream(rootLogger.getHandlers())
+          .forEach(
```

Comment:
I don't think it is required since we are not registering any handlers but again having the code is not a problem. Looks fine to me.